src/test/java/org/broadinstitute/consent/http/db/DacDAOTest.java

```diff
@@ -96,12 +97,14 @@ void testFindAllNoDatasets() {
     Integer dacId2 = createRandomDAC();
     List<Dac> dacs = dacDAO.findAll();
 
-    Dac dac1 = dacs.get(0);
+    Dac dac1 = dacs.stream().filter(d -> d.getDacId().equals(dacId1)).findFirst().orElse(null);
+    assertNotNull(dac1);
```

Comment:
I would use `orElseThrow()` instead of `assertNotNull()`; either way the test fails with an exception. It's true that one is an NPE and one is an `AssertionError`.. I don't mind this difference though.
```suggestion
    Dac dac1 = dacs.stream().filter(d -> d.getDacId().equals(dacId1)).findFirst().orElseThrow();
```

Reply:
Nice simplification. We're pretty sure the `get(0)` calls were the root of the flakiness since the DAO methods do not have an enforced order.